manual/de/Help_ch_Account-Actions.xml

```diff
@@ -1376,11 +1459,24 @@ and update the appendix. -->
         Konten neu eröffnen, können Sie hier den Abschlusssaldo in die neue Periode übertragen.
       </para>
 
+      <figure pgwide="1">
+        <title>Neues Konto erstellen, Reiter Anfangsbestand</title><screenshot>
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="figures/AcctCreate_OpenBal.png" 
+                         srccredit="Christian Wehling"></imagedata>
+            </imageobject>
+          </mediaobject>
+        </screenshot>
+      </figure>
+
       <para>Wenn Sie ihre &app;-Datei über einen längeren Zeitraum fortlaufend nutzen, können Sie mit Hilfe
-        des <xref linkend="tool-close-book" /> (<menuchoice>
+        des <xref linkend="tool-close-book" /> (
+        <menuchoice>
           <guimenu><accel>W</accel>erkzeug</guimenu><guimenuitem>Buch<accel>a</accel>bschluss</guimenuitem>
-        </menuchoice>) den Anfangsbestand automatisch berechnen lassen. Hierbei ist der Abschlußsaldo der vorherigen
-        Periode der Anfangsbestand der neuen Periode.
+        </menuchoice>
+        ) den Anfangsbestand automatisch berechnen lassen. Hierbei ist der Abschlußsaldo der
+        vorherigen Periode der Anfangsbestand der neuen Periode.
       </para>
```

Comment:
in 1490f: `<guilabel>Saldo-
        Informationen</guilabel>`
Fehler beim Tippen oder in xmlformat?

Reply:
Tippfehler, da war ein Zeilenumbruch im xml-Element, hat mit `xmlformat` nichts zu tun.